features/git-rename-branch/destination_branch_exists_locally.feature

```diff
@@ -0,0 +1,36 @@
+Feature: git rename-branch: errors when the destination branch exists locally
+
+  As a developer trying to rename a branch into an already existing branch
+  I want the command to abort with an error message
+  So that I don't lose work by accidentally overwriting existing branches
```

Comment:
Please add a dot at the end, since this is an (admittedly pretty lengthy and cumbersome) sentence.